Toolchain/BuildGNU.sh

```diff
@@ -299,7 +299,7 @@ pushd "$DIR/Build/$ARCH"
                                             --disable-nls \
                                             --disable-libstdcxx-pch \
                                             --enable-shared \
-                                            --enable-languages=c,c++ \
+                                            --enable-languages=c,c++,objc,obj-c++ \
```

Comment:
This only enables objective C and C++ for gcc, is it already enabled for the clang toolchain?

Reply:
Clang always includes all supported languages.

Reply:
This is a bit cursed, but works:
```sh
❯ export SDKROOT=$(xcrun --show-sdk-path --sdk macosx)
❯ cat test.m
#import <Foundation/Foundation.h>

int main(int argc, const char * argv[]) {
    @autoreleasepool {
        // insert code here...
        NSLog(@"Hello, World!");
    }
    return 0;
}
❯ Toolchain/Local/clang/bin/clang --target=arm64-apple-macos14 -framework Foundation test.m
❯ ./a.out
2023-07-27 20:52:10.480 a.out[34111:641191] Hello, World!
```